tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ZepplinResource.java

```diff
@@ -63,7 +73,11 @@ public Response headZepplin(@RequestBody(ref = "#/paths/~1zepplins~1{id}/delete/
     @Operation(summary = "Deprecate outdated airship technology", operationId = "deprecateZepplin")
     @Produces("text/plain")
     @SecurityRequirement(name = "mutualTLSScheme", scopes = "zepplinScope")
-    public Response getZepplin(@RequestBody(ref = "#/paths/~1zepplins~1{id}/delete/requestBody") String string) {
+    @Parameters(value = {@Parameter(name = "string", description = "something about a string", in = ParameterIn.QUERY,
+                                    required = true,
+                                    schema = @Schema(type = SchemaType.OBJECT), style = ParameterStyle.PIPEDELIMITED)})
```

Comment:
`@Parameter` is a repeatable annotation. You can just use it on its own, rather than wrapping it in `@Parameters`.